joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 125  

left_arm_joint_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - left_shoulder_1_joint
    - left_shoulder_2_joint
    - left_shoulder_3_joint
    - left_elbow_1_joint
    - left_elbow_2_joint
    - left_wrist_1_joint
    - left_wrist_2_joint
  constraints:
    goal_time: 0.6
    stopped_velocity_tolerance: 0.05
    left_shoulder_1_joint: {trajectory: 0.1, goal: 0.1}
    left_shoulder_2_joint: {trajectory: 0.1, goal: 0.1}
    left_shoulder_3_joint: {trajectory: 0.1, goal: 0.1}
    left_elbow_1_joint: {trajectory: 0.1, goal: 0.1}
    left_elbow_2_joint: {trajectory: 0.1, goal: 0.1}
    left_wrist_1_joint: {trajectory: 0.1, goal: 0.1}
    left_wrist_2_joint: {trajectory: 0.1, goal: 0.1}      
  stop_trajectory_duration: 0.5
  state_publish_rate:  25
  action_monitor_rate: 10

right_arm_joint_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - right_shoulder_1_joint
    - right_shoulder_2_joint
    - right_shoulder_3_joint
    - right_elbow_1_joint
    - right_elbow_2_joint
    - right_wrist_1_joint
    - right_wrist_2_joint
  constraints:
    goal_time: 0.6
    stopped_velocity_tolerance: 0.05
    right_shoulder_1_joint: {trajectory: 0.1, goal: 0.1}
    right_shoulder_2_joint: {trajectory: 0.1, goal: 0.1}
    right_shoulder_3_joint: {trajectory: 0.1, goal: 0.1}
    right_elbow_1_joint: {trajectory: 0.1, goal: 0.1}
    right_elbow_2_joint: {trajectory: 0.1, goal: 0.1}
    right_wrist_1_joint: {trajectory: 0.1, goal: 0.1}
    right_wrist_2_joint: {trajectory: 0.1, goal: 0.1}      
  stop_trajectory_duration: 0.5
  state_publish_rate:  25
  action_monitor_rate: 10
